examples/oxr/python/test_oak_camera.py

```
@@ -85,7 +85,7 @@ def _run_schema_pusher(
                 start_time = time.time()
                 frame_count = 0
                 last_print_time = 0
-                last_device_times = {n: -1 for n in stream_names}
+                last_seq = {n: -1 for n in stream_names}
```

Ruff (0.15.2)

[convention] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)